### DISCUSSION TITLE
Decide the nature of the menu and footer components and extend it when required

### DISCUSSION THREAD

**Discussion by delapuente** [Qiskit/qiskit.org#2387] [General]
Decide if the current `AppFooter` and `TheMenu` are [Single Instance components](https://vuejs.org/v2/style-guide/#Single-instance-component-names-strongly-recommended) or [Base components](https://vuejs.org/v2/style-guide/#Base-component-names-strongly-recommended) and rename accordingly. Extend this pattern where it is needed.

**Comment by delapuente** [comment]
From the [definition given in Vue docs](https://vuejs.org/v2/guide/components-registration.html#Automatic-Global-Registration-of-Base-Components), I follow that base components are relatively simple, like small wrappers. I don't think the site footer and menu are simple components at all.

However, they do fall under the [_once per page_ category](https://vuejs.org/v2/style-guide/#Single-instance-component-names-strongly-recommended) so I think they should be singleton components prefixed with `The`.

Thoughts @y4izus?

**Comment by y4izus** [comment]
Agree
